pydicom/tests/test_encaps.py

```diff
@@ -0,0 +1,776 @@
+# Copyright 2008-2017 pydicom authors. See LICENSE file for details.
+"""Test for encaps.py"""
+
+import pytest
+
+from pydicom.encaps import (generate_pixel_data_fragment, get_frame_offsets,
+                            generate_pixel_data_frame, generate_pixel_data,
+                            decode_data_sequence, defragment_data, read_item)
+from pydicom.filebase import DicomBytesIO
+
+
+def assert_raises_regex(type_error, message, func, *args, **kwargs):
+    """Taken from https://github.com/glemaitre/specio, BSD 3 license."""
+    with pytest.raises(type_error) as excinfo:
+        func(*args, **kwargs)
+    excinfo.match(message)
+
+
+class TestGetFrameOffsets(object):
+    """Test encaps.get_frame_offsets"""
+    def test_bad_tag(self):
+        """Test raises exception if no item tag."""
+        # (fffe,e100)
+        bytestream = b'\xFE\xFF\x00\xE1' \
+                     b'\x08\x00\x00\x00' \
+                     b'\x01\x02\x03\x04\x05\x06\x07\x08'
+        fp = DicomBytesIO(bytestream)
+        fp.is_little_endian = True
+        # For some reason matching the messages fails using assert_raises_regex
+        # It looks like the conversion of the tag to str is the issue
```

Comment:
I guess I should've figured it out from the name, needed to escape the brackets.